app/utils/database.py

```diff
@@ -286,7 +296,54 @@ class User(BaseModel):
     uuid = UUIDField()
     password_hash = CharField(128)
     validated = BooleanField(default=False)
-    privileges = TextField(null=True)
+    _privileges = TextField(column_name="privileges", default='{"roles":[]}')
```

Comment:
I found this naming scheme and the database usage interesting. Let's look at it.
First, what other privs than `roles` are you expecting?
Secondly, you store it as a json string in the DB. I don't have a particularily strong objection about this but how come you aren't storing in a row-oriented/SQL-fashion instead?

Reply:
When we talked about it in chat this was the solution people seemed to favor (while I was slightly more in of having permissions in tables). The privileges field was already present in the database, so I used it... :)